cmd/health_alive.go

```diff
@@ -28,7 +28,7 @@ Note:
 	Run: func(cmd *cobra.Command, _ []string) {
 		client := newClient(cmd)
 
-		r, err := client.API.IsInstanceAlive(api.NewIsInstanceAliveParams())
+		r, err := client.Health.IsInstanceAlive(health.NewIsInstanceAliveParams())
```

Comment:
I honestly don't know why the SDK changed, but :shrug:

Reply:
I think it comes from changes in `github.com/ory/x/healthx`. Fine with me.